packages/db/src/schema/workspaces/workspace.ts

```diff
@@ -36,3 +36,18 @@ export const workspaceRelations = relations(workspace, ({ many }) => ({
   usersToWorkspaces: many(usersToWorkspaces),
   pages: many(page),
 }));
+
+export const application = sqliteTable("application", {
+  id: integer("id").primaryKey(),
+  name: text("name"), // friendly name for the project
+  dsn: text("dsn").unique(), // dsn for the source
+
+  workspaceId: integer("workspace_id").references(() => workspace.id),
+
+  createdAt: integer("created_at", { mode: "timestamp" }).default(
+    sql`(strftime('%s', 'now'))`,
+  ),
+  updatedAt: integer("updated_at", { mode: "timestamp" }).default(
+    sql`(strftime('%s', 'now'))`,
+  ),
+});
```

Comment:
Let's create a `/schema/applications/application.ts` dir with the table schema to keep the structure.